app/services/audit/catalog_to_moab.rb

```diff
@@ -7,7 +7,7 @@ class CatalogToMoab
 
     def initialize(moab_record)
       @moab_record = moab_record
-      @logger = Logger.new(Rails.root.join('log', 'c2m.log'))
+      @logger = Logger.new(Rails.root.join('log', 'audit_checksum_validation.log'))
```

Comment:
i think this one should change to e.g. `audit_catalog_to_moab.log`

Reply:
@jmartin-sul fixed. Thanks :-)